support-pp.js

```diff
@@ -409,10 +409,10 @@ registerPlugin(
 
       //                         -> Ticket
       {
-        name: "spTicketActiv",
+        name: "spTicketActive",
         indent: 2,
         title:
-          "[TicketSystem] TicketSystem Advanced and Minimal. Manage your Tickets with a webpanal... [NEW] ",
+          "[TicketSystem] TicketSystem Advanced and Minimal. Manage your Tickets with a web panel... [NEW] ",
```

Comment:
```suggestion
          "[TicketSystem] TicketSystem Advanced and Minimal. Manage your Tickets with a web panel... [WIP] ",
```
To avoid questions about the panel